DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: RPresto
 Title: DBI Connector to Presto
-Version: 1.3.8
+Version: 1.3.8.9000
```

Comment:
The tibble change is possibly breaking factor behavior, so we should bump to 1.4.0.

Reply:
Good point and Done. I've considered making `tibble` output an opt-in option for users so to maximize backward-compatibility. But I think the benefits of having a `tibble` output outweigh the costs. Before we release 1.4.0, I will make sure that the change of API is well communicated in NEWS.md and vignettes. I don't suppose many users will be affected by this because `tibble` behaves like a `data.frame` in majority of the cases. But at least if users get backward-compatibility issues, they can easily find out why in our documentations.